dataactbroker/routes/dashboard_routes.py

```diff
@@ -86,6 +87,24 @@ def get_active_submission_overview(submission, file, **kwargs):
         error_level = kwargs.get('error_level')
         return active_submission_overview(submission, file, error_level)
 
+    @app.route("/v1/get_impact_counts/", methods=["GET"])
+    @convert_to_submission_id
+    @requires_submission_perms('reader')
+    @use_kwargs({
+        'file': webargs_fields.String(validate=webargs_validate.
+                                      OneOf(['A', 'B', 'C', 'cross-AB', 'cross-BC', 'cross-CD1', 'cross-CD2'],
+                                            error='Must be A, B, C, cross-AB, cross-BC, cross-CD1, or cross-CD2'),
+                                      required=True),
+        'error_level': webargs_fields.String(validate=webargs_validate.
+                                             OneOf(['warning', 'error', 'mixed'],
+                                                   error='Must be either warning, error, or mixed'),
+                                             missing='warning')
+    })
+    def impact_counts(submission, file, **kwargs):
+        """ Returns an overview of the requested submission for the active dashboard """
```

Comment:
`Returns the impact counts of the requested submission for the active dashboard` ?